tests/foreman/ui/test_settings.py

```diff
@@ -478,6 +476,24 @@ def test_negative_update_hostname_with_empty_fact():
 
     :expectedresults: Error should be raised on setting empty value for
         hostname_facts setting
-
-    :CaseAutomation: notautomated
     """
+    property_name = "Discovered"
+    discovery_config_default_param = { "discovery_hostname" : "" }
+    discovery_config_default_param = {
+        content: entities.Setting().search(query={'search': f'name='
+                                                            f'{content}'})[
+            0] for content in discovery_config_default_param
+    }
+    discovery_config_new_param = { "discovery_hostname" : ""}
+    with session:
+        try:
+            for discovery_content, discovery_content_value in \
+                    discovery_config_new_param.items():
```

Comment:
I'd prefer to have shorter variable names and not use backslash:
```
for param, param_value in discovery_config_new_param.items():
    discovery_update_response = session.settings.update(param, param_value)
```

Reply:
Variables replaced with shorter names.